types/hooks/screens/useAnonymousChannelListScreenHook.types.ts

```diff
@@ -1,10 +1,11 @@
 import {ChannelList} from '../../database/schema/ChannelList.types';
+import {ContactScreenPayload} from './useChatUtilsHook.types';
 
 interface UseAnonymousChannelListScreenHook {
   channels: ChannelList[];
   goToChatScreen: (channel: ChannelList) => void;
   goToPostDetailScreen: (channel: ChannelList) => void;
-  goToContactScreen: () => void;
+  goToContactScreen: ({from}: ContactScreenPayload) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the `goToContactScreen` function signature is a good improvement as it allows more flexibility by accepting a payload object. However, please ensure that all usages of this function throughout the codebase are updated to match this new signature to avoid potential runtime errors.
```diff
-  goToContactScreen: () => void;
+  goToContactScreen: ({from}: ContactScreenPayload) => void;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->